trunk8.js

```diff
@@ -8,7 +8,15 @@
  * Date: September 26, 2012
  */
 
-(function ($) {
+(function (root, factory) {
```

Comment:
It's UMD https://github.com/umdjs/umd
On 29 Mar 2016 20:17, "Rick Viscomi" <notifications@github.com> wrote:

> In trunk8.js
> <https://github.com/rviscomi/trunk8/pull/72#discussion_r57783314>:
>
> > @@ -8,7 +8,15 @@
> >   * Date: September 26, 2012
> >   */
> >
> > -(function ($) {
> > +(function (root, factory) {
>
> I'm not familiar with this pattern. Do you have a link to some docs
> describing what this does/is for?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rviscomi/trunk8/pull/72/files/8e76f19a58e52201588e1bfbc554b82f659a5a7f#r57783314>
>